src/components/pages/ParentDashFaq/index.js

```diff
@@ -0,0 +1 @@
+export { default as ParentDashFaq } from './ParentDashFaqContainer';
```

Comment:
good stuff - keeping it consistent with the rest of the file structure of the application.